test/bridge/testhorton.py

```diff
@@ -107,7 +107,34 @@ def test_makehorton(self):
                     )
 
         elif self._hortonver == 3:
```

Comment:
Can you please make a separate test class for 2 and 3 versions?

Reply:
I've separated horton 2 and 3 test classes. Thanks!